languages.yaml

```
@@ -138,6 +138,35 @@ Assembly:
       - yellow
       - green
     chip: "#6E4C13"
+Astro:
+  type: programming
+  ascii: |                                               
+    {0}                                                       
+    {0}                      *@@@@@@@@@#                      
+    {0}                      @@@@@@@@@@@#                     
+    {0}                     %@@@@@@@@@@@@                     
+    {0}                    #@@@@@@#@@@@@@%                    
+    {0}                    @@@@@@# *@@@@@@*                   
+    {0}                   %@@@@@@    %@@@@@@                   
+    {0}                  *@@@@@@*    @@@@@@#                  
+    {0}                  @@@@@@%      #@@@@@@                  
+    {0}                 #@@@@@@*      @@@@@@%                 
+    {0}                 @@@@@@#       *@@@@@@*                
+    {0}                %@@@%%#         ##%@@@@                
+    {0}                #*                   *#*              
+    {1}                      #*        **                     
+    {1}                      %%%%%%%%%%%*                    
+    {1}                      *#  #%%%%#*                     
+    {1}                          *%%#                        
+    {1}                           ##  
```

Looks like the ASCII art exceeds our maximum width, but I believe that can be fixed by just removing the trailing spaces.
```suggestion
    {0}
    {0}                      *@@@@@@@@@#
    {0}                      @@@@@@@@@@@#
    {0}                     %@@@@@@@@@@@@
    {0}                    #@@@@@@#@@@@@@%
    {0}                    @@@@@@# *@@@@@@*
    {0}                   %@@@@@@    %@@@@@@
    {0}                  *@@@@@@*    @@@@@@#
    {0}                  @@@@@@%      #@@@@@@
    {0}                 #@@@@@@*      @@@@@@%
    {0}                 @@@@@@#       *@@@@@@*
    {0}                %@@@%%#         ##%@@@@
    {0}                #*                   *#*
    {1}                      #*        **
    {1}                      %%%%%%%%%%%*
    {1}                      *#  #%%%%#*
    {1}                          *%%#
    {1}                           ##
```